runbooks/runbook.md

```diff
@@ -0,0 +1,118 @@
+# UPP - Content Preview
+
+Content Preview accesses unpublished content and previews it before publishing.
+
+## Code
+
+content-preview
+
+## Primary URL
+
+https://upp-prod-delivery-glb.upp.ft.com/__content-preview/
+
+## Service Tier
+
+Bronze
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- dimitar.terziev
+- elitsa.pavlova
+- kalin.arsov
+- hristo.georgiev
+- tsvetan.dimitrov
+- elina.kaneva
+- robert.marinov
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+The Content Preview service serves published and unpublished content from Methode CMS so that journalists can preview unpublished articles in the context that closely matches the live publication on the Next FT platform. 
+It is one of internal services that comprise UPP read stack and is not meant to be accessible publicly but to collaborate with other services such as `content-public-read-preview`, which in turn gives access to an authenticated public API to read preview content. 
+Content Preview relies on Methode API service and Methode Article Transformer service; both underlying services are crucial to serve preview content.
+
+## Contains Personal Data
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Dependencies
+
+- upp-neo4j-cluster
```

Comment:
Neo4j is not a dependancy here, but both articles mappers and Methode API are:
- <https://biz-ops.in.ft.com/System/up-mam>
- <https://biz-ops.in.ft.com/System/up-maicm>
- <https://biz-ops.in.ft.com/System/up-mapi>

Reply:
Copy-paste mistake, I'll fix it.